### PR TITLE
Unit tests refactoring

### PR DESCRIPTION
**Pull request by nikmel2803**

### CODE REVIEW

File: lib/broker/broker.go [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+	)
+	return receiving, err
+}
+
+func (b *RabbitMQ) Publish(queueName string, payload []byte) error {
+	err := b.channel.PublishWithContext(
+		context.TODO(),
+		"",
+		queueName,
+		false,
```
Comment (n0str) [inline lib/broker/broker.go]: please, comment this

**Review by n0str — APPROVED** [review]

File: go.work.sum [hunk 2]
```diff
@@ -0,0 +1,4 @@
+github.com/stretchr/objx v0.4.0/go.mod h1:YvHI0jy2hoMjB+UWwv71VJQ9isScKT/TqJzVSSt89Yw=
```
Comment (thomasfire) [inline go.work.sum:1]: Seems like a packet manager cache

File: lib/broker/go.sum [hunk 3]
```diff
@@ -0,0 +1,40 @@
+github.com/davecgh/go-spew v1.1.0/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
```
Comment (thomasfire) [inline lib/broker/go.sum:1]: same cache-like

File: lib/worker/go.sum [hunk 4]
```diff
@@ -0,0 +1,52 @@
+github.com/benbjohnson/clock v1.1.0 h1:Q92kusRqC1XV2MjkWETPvjJVqKetz1OzxZB7mHJLju8=
```
Comment (thomasfire) [inline lib/worker/go.sum:1]: same cache

File: lib/worker/worker.go [hunk 5]
```diff
@@ -131,13 +84,27 @@ func New(rabbitmqURL string, queueName string, handler TaskHandler) *Worker {
...
 
-// CreateDefaultLogger creates default logger with a certain log level
-func CreateDefaultLogger(level zapcore.Level) *zap.SugaredLogger {
+// CreateDefaultLogger creates default logger
+func CreateDefaultLogger() *zap.SugaredLogger {
+	return CreateDefaultLoggerWithLevel(zapcore.InfoLevel)
+}
+
+// CreateTestLogger creates logger for testing purposes
+func CreateTestLogger() *zap.SugaredLogger {
```
Comment (thomasfire) [inline lib/worker/worker.go:99]: doubt it should be in the lib part

File: mocks/task_handler.go [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+)
+
+// TaskHandler is an autogenerated mock type for the TaskHandler type
+type TaskHandler struct {
+	mock.Mock
+}
+
+// Execute provides a mock function with given fields: ctx
+func (_m *TaskHandler) Execute(ctx worker.HandlerContext) error {
+	ret := _m.Called(ctx)
```
Comment (thomasfire) [inline mocks/task_handler.go:17]: wtf

**Review by thomasfire — CHANGES_REQUESTED** [review]

**Review by thomasfire — APPROVED** [review]
ничего не понятно, но очень интересно

**Pull request merged**